src/callbacks/wandb_callbacks.py

```diff
@@ -41,18 +51,38 @@ def on_train_start(self, trainer, pl_module):
 
 
 class UploadCodeAsArtifact(Callback):
-    """Upload all *.py files to wandb as an artifact, at the beginning of the run."""
+    """Upload all code files to wandb as an artifact, at the beginning of the run."""
+
+    def __init__(self, code_dir: str, use_git: bool = True):
+        """
 
-    def __init__(self, code_dir: str):
+        :param code_dir:
```

Comment:
Please follow the google docstring convention as it's already used in `utils.py`

Reply:
This is solved